rules/windows/command_and_control_sunburst_c2_activity_detected.toml

```diff
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2020/12/14"
+maturity = "production"
+updated_date = "2020/12/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+The malware known as SUNBURST targets the SolarWind's Orion business software for command and control. This rule detects
+post-exploitation command and control activity of the SUNBURST backdoor.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "SUNBURST Command and Control Activity"
+note = "The SUNBURST malware attempts to hide within the Orion Improvement Program (OIP) network traffic. As this rule detects post-exploitation network traffic, investigations into this should be prioritized."
+references = [
+    "https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html",
+]
+risk_score = 73
+rule_id = "22599847-5d13-48cb-8872-5796fee8692b"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control"]
+type = "query"
+
+query = '''
+event.category:network and event.type:protocol and network.protocol:http and process.name:("SolarWinds.BusinessLayerHostx64.exe" or "ConfigurationWizard.exe" or "NetflowDatabaseMaintenance.exe" or "NetFlowService.exe" or "SolarWinds.Administration.exe" or "SolarWinds.BusinessLayerHost.exe" or "SolarWinds.Collector.Service.exe" or "SolarwindsDiagnostics.exe") and (http.request.body.content:(*/swip/Upload.ashx* and (POST* or PUT*)) or (http.request.body.content:(*/swip/SystemDescription* and (GET* or HEAD*))) or (http.request.body.content:(*/swip/Events* and (GET* or HEAD*)))) and not http.request.body.content:*solarwinds.com*
```

Comment:
Do we know the resource impact for all the leading/trailing wildcards?

We could probably make this a bit more readable too, something like below - 
```
event.category:network and event.type:protocol and network.protocol:http and 
	process.name:(
		"SolarWinds.BusinessLayerHostx64.exe" or 
		"ConfigurationWizard.exe" or 
		"NetflowDatabaseMaintenance.exe" or 
		"NetFlowService.exe" or 
		"SolarWinds.Administration.exe" or 
		"SolarWinds.BusinessLayerHost.exe" or 
		"SolarWinds.Collector.Service.exe" or 
		"SolarwindsDiagnostics.exe"
		) and 
	(http.request.body.content:(*/swip/Upload.ashx* and (POST* or PUT*)) or 
	(http.request.body.content:(*/swip/SystemDescription* and (GET* or HEAD*))) or 
	(http.request.body.content:(*/swip/Events* and (GET* or HEAD*)))) and not 
	http.request.body.content:*solarwinds.com*
```

Reply:
Would definitley like a more elegant approach, but having to carve out of the `http.request.body.content`, it's kind of smashed together (example traffic, not from SUNBURST):
```
GET /8SE/44?MI=760C4DFD5ADCA7CE12EF519D059E335F&LV=1.0.3&OS=WindowsNT10.0;Win64;x64&TE=37&TV=isU602%7CpkMicrosoftBingQuickSearch%7Ctmen-US%7CbvChrome87.0.4280.88%7Cexekccfocemkpmljfcnhhlinkeafbeocco%7Ces2%7Cmu27BC5E97AA8F69D036DB51D6AB27686C%7Cchqsanaheim HTTP/1.1
Host: g.ceipmsn.com
Connection: keep-alive
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36 Edg/87.0.664.60
Accept: */*
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.9
```
Complicated by us not having example data to test with to see _exactly_ how the URI is structured.

Updated the query to be more readable. Great idea.